src/main/java/com/zendesk/maxwell/schema/columndef/DateTimeColumnDef.java

```diff
@@ -25,24 +25,34 @@ private static SimpleDateFormat getDateTimeFormatter() {
 	public boolean matchesMysqlType(int type) {
 		if ( getType().equals("datetime") ) {
 			return type == MySQLConstants.TYPE_DATETIME ||
-				   type == MySQLConstants.TYPE_DATETIME2;
+				type == MySQLConstants.TYPE_DATETIME2;
 		} else {
 			return type == MySQLConstants.TYPE_TIMESTAMP ||
-				   type == MySQLConstants.TYPE_TIMESTAMP2;
+				type == MySQLConstants.TYPE_TIMESTAMP2;
 		}
 	}
 
 	private String formatValue(Object value) {
 		/* protect against multithreaded access of static dateTimeFormatter */
 		synchronized ( DateTimeColumnDef.class ) {
-			if ( value instanceof Long && getType().equals("datetime") )
+			if ( value instanceof Long && getType().equals("datetime") ) {
 				return formatLong(( Long ) value);
-			else if ( value instanceof Timestamp )
-				return getDateTimeFormatter().format(( Timestamp ) value);
-			else if ( value instanceof Date )
+			} else if ( value instanceof Timestamp ) {
+				int nanos = (( Timestamp ) value).getNanos();
+				if ( nanos > 0 ) {
+					StringBuilder result = new StringBuilder();
+					result.append(getDateTimeFormatter().format(( Date ) value));
+					result.append(".");
+					result.append(String.format("%06d", nanos/1000));
```

Comment:
is it possible to capture the resolution that the user specified at table-creation time?  I can help out with capturing it from a `CREATE TABLE` statement, but wondering if it's possible to capture it from INFORMATION_SCHEMA